src/assets/toolkit/styles/vendor/wordpress.css

```diff
@@ -0,0 +1,92 @@
+/* Wordpress generated classes */
+
+.aligncenter,
+div.aligncenter,
+a img.aligncenter {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.alignright,
+a img.alignright {
+  float:right;
+  margin-left: var(--space-sm);
+  margin-bottom: var(--space-sm);
+}
+
+.alignleft,
+a img.alignleft {
+  float: left;
+  margin-right: var(--space-sm);
+  margin-bottom: var(--space-sm);
+}
+
+/* Image scaling
+ * Make sure images with WordPress-added height and width
+ * attributes are scaled correctly
+ */
+
+img[class*="align"],
+img[class*="wp-image-"] {
+  height: auto;
```

Comment:
This was something that I had tried with Ceasefire, that seemed to be working at first, and then later not, for...reasons. Instead I resolved with CSS. I'm open to trying that path again, though.